sites/game-client/ui/user-overlay.mjs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2021 HITCON Online Contributors
+// SPDX-License-Identifier: BSD-2-Clause
+
+import Overlay from './overlay.mjs';
+
+const USER_OVERLAY_DIV = 'user-overlay';
+
+class UserOverlay extends Overlay {
+    /**
+   * @constructor
+   * @param {MainUI} mainUI - The MainUI object.
+   */
+  constructor(mainUI) {
+    const dom = document.getElementById(USER_OVERLAY_DIV);
+    super(mainUI, dom);
+  }
+
+  show() {
+    $(this.dom).show();
```

Comment:
Maybe add a TODO on correctly implementing the Overlay. i.e. add corresponding OverlayPosition

Reply:
added.